go.mod

```diff
@@ -3,27 +3,21 @@ module github.com/projectcalico/app-policy
 go 1.12
```

Comment:
isn't our go-build image up to go 1.15?

Reply:
Yeah, I thought I updated this but I guess I didn't